.storybook/setupTwig.js

```diff
@@ -3,13 +3,46 @@ const twigDrupal = require('twig-drupal-filters');
 const twigBEM = require('bem-twig-extension');
 const twigAddAttributes = require('add-attributes-twig-extension');
 
-module.exports.namespaces = {
-  atoms: resolve(__dirname, '../', 'components/01-atoms'),
-  molecules: resolve(__dirname, '../', 'components/02-molecules'),
-  organisms: resolve(__dirname, '../', 'components/03-organisms'),
-  templates: resolve(__dirname, '../', 'components/04-templates'),
+/**
+ * Fetches project-based variant configuration. If no such configuration
+ * exists, returns default values.
+ *
+ * @returns project-based variant configuration, or default config.
+ */
+const fetchVariantConfig = () => {
+  try {
+    return require('../project.emulsify.json').variant.structureImplementations;
```

Comment:
@patrickocoffeyo So how does project.emulsify.json come into play here, it seems like all of this data would have been stored in the system.emulsify.json file so how does it get here?  Copy paste from the system file?

Reply:
@ccjjmartin great question!

Determining where `system.emulsify.json` lives is a bit of a task. It's not stored in the project, but rather `emulsify-cli` caches it in your home folder (along with the rest of the system). In the future, when we support variants that _don't_ live in the same repository as the system, this will become even more complex. So, in order for us to read this configuration from that cached file, we essentially need to publish the `emulsify-cli` project as a library so `emulsify-drupal` (and other starters) can make use of [this function](https://github.com/emulsify-ds/emulsify-cli/blob/develop/src/util/cache/getJsonFromCachedFile.ts), as well as some other utils.

That's something I'll be tackling next quarter. For now, I've adjusted `emulsify-cli` so that it writes variant configuration to the project's `project.emulsify.json` file.

Reply:
Ok, I didn't test the functional here because I am already running two migration scripts right now but this looks like a :shipit: if `yarn develop` is working for you locally.

Reply:
@ccjjmartin Right on, I say we hold off on merging this until we fix the `eslint-webpack-plugin` issue. I'd like to get @ModulesUnraveled to poke at this functionally and make sure it doesn't create any unintended side effects, especially since none of this is unit tested.